src/interfaces/IRegistryCoordinator.sol

```diff
@@ -1,18 +1,29 @@
 // SPDX-License-Identifier: BUSL-1.1
 pragma solidity =0.8.12;
 
+import {IBLSApkRegistry} from "src/interfaces/IBLSApkRegistry.sol";
+import {IStakeRegistry} from "src/interfaces/IStakeRegistry.sol";
+import {IIndexRegistry} from "src/interfaces/IIndexRegistry.sol";
+
 /**
  * @title Interface for a contract that coordinates between various registries for an AVS.
  * @author Layr Labs, Inc.
  */
 interface IRegistryCoordinator {
     // EVENTS
+
     /// Emits when an operator is registered
     event OperatorRegistered(address indexed operator, bytes32 indexed operatorId);
 
     /// Emits when an operator is deregistered
     event OperatorDeregistered(address indexed operator, bytes32 indexed operatorId);
-    
+
+    event OperatorSetParamsUpdated(uint8 indexed quorumNumber, OperatorSetParam operatorSetParams);
+
+    event ChurnApproverUpdated(address prevChurnApprover, address newChurnApprover);
+
+    event EjectorUpdated(address prevEjector, address newEjector);
```

Comment:
ahhh, so the key change being made here is merging the `IBLSRegistryCoordinatorWithIndices` interface _into the existing_ `IRegistryCoordinator` interface.
Not sure if I understood that to be an intention of the PR from our discussion.
I'll review the state of the repo to better understand if I agree with this particular move.

Reply:
On review, I'm OK with this. Seems the things in the interface aren't at all BLS-specific, other than the `blsApkRegistry`, and that's super reasonable to change in the future.

Reply:
Definitely wasn't the intent, but seemed the easiest way to do things.

It's not a flawless BLS-neutral interface, but as you said, we can fix that later.